src/Database/Commands/PatchByIndexCommand.ts

```diff
@@ -29,17 +29,16 @@ export class PatchByIndexCommand extends IndexQueryBasedCommand {
   }
 
   public setResponse(response: IResponse): IRavenResponse | IRavenResponse[] | void {
-    const responseBody: IResponseBody = response.body;
-    const status: StatusCode = response.statusCode;
+    const result: IRavenResponse = <IRavenResponse>super.setResponse(response);
 
-    if(!responseBody) {
-      throw new ErrorResponseException(StringUtil.format('Could not find index {0}', this.indexName));
+    if(!response.body) {
+      throw new IndexDoesNotExistException(StringUtil.format('Could not find index {0}', this.indexName));
     }
 
-    if (![StatusCodes.Ok, StatusCodes.Accepted].includes(status)) {
+    if (![StatusCodes.Ok, StatusCodes.Accepted].includes(response.statusCode)) {
```

Comment:
I noticed you use vscode. I get the following error there. Is there something wrong with my setup or typings?

file: 'file:///c%3A/work/ravendb-nodejs-client/src/Database/Commands/PatchByIndexCommand.ts'
severity: 'Error'
message: 'Property 'includes' does not exist on type 'StatusCode[]'.'
at: '38,49'
source: 'ts'